### ISSUE TITLE
manage_cases(): StartTag: invalid element name

### ISSUE THREAD

**Issue by mtalluto** [r-lib/vdiffr#94]
When I run manage_cases() on a new figure, I am getting a warning on the console as follows:

Warning: Error in read_xml.raw: StartTag: invalid element name [68]
  103: read_xml.raw
  102: read_xml.character
  100: .f
   99: map
   98: widget
   96: func
   83: origRenderFunc
   82: output$slide
    2: shiny::runApp
    1: manage_cases

Moreover, the new plot shows up as a broken image icon. A file is created in figs/plotting, but it's just an image of the error:

<img width="1020" alt="Screenshot 2020-10-22 at 15 04 11" src="https://user-images.githubusercontent.com/5796287/96875742-e1fc3c80-1477-11eb-93b2-b0e85cb9a62d.png">

This is only affecting plots using ggplot2; if I create a new plot in base graphics, it works fine. I also tried deleting figs, anything using base graphics is fine, anything using ggplot2 breaks. Any ideas what could be causing this? This is on ggplot2 v3.3.2, vdiffr 0.3.3, RStudio 1.3.959, R 4.0.1. Using the base R GUI or RStudio makes no difference.

**Comment by lionel-** [comment]
What do you get when you try to plot with svglite?

**Comment by mtalluto** [comment]
Works perfectly with svglite

**Comment by lionel-** [comment]
Might be fixed when we next update the copy of svglite that is embedded in vdiffr then.

But I'll need a minimal and reproducible example so I can make sure.

**Comment by pat-s** [comment]
Here's a reprex for a `plotly` object:

```r
test_that("test", {
  vdiffr::expect_doppelganger(
    "test plotly",
    plotly::plot_ly())
})
```

I've seen there were multiple issues/PRs related to `plotly` already, not sure what the current status/plan is.

**Comment by lionel-** [comment]
Should be fixed by #106 which updated the embedded copy of svglite.
